### DISCUSSION TITLE
Add fields to image block?

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [johannschopplich/nuxt-kql#21] [Q&A]
Hello,
Is there a correct way to add additional field to the image block? I have added the to the image blueprint file in kirby, but they don't show up in the block when adding an image to the page

**Comment by johannschopplich** [comment]
Of course. But this needs to be configured in the backend. For all fields you need, you have to:
- either [configure the default image resolver](https://kirby.tools/docs/headless/field-methods#files-and-pages-resolver)
- or use a [custom resolver for that specific block](https://kirby.tools/docs/headless/field-methods#custom-resolvers)

**Reply by pixelpaulaus** [reply]
but isnt that when you want to output the block? The custom fields are not even showing up in the kirby editor
